yt/yt/ytlib/api/native/client_operation_info_impl.cpp

```diff
@@ -699,6 +702,22 @@ TOperation TClient::DoGetOperationImpl(
     return *archiveResult;
 }
 
+static std::optional<TAccessControlRule> GetOperationAccessControlRule(const TOperation& operation)
+{
+    if (operation.RuntimeParameters) {
+        auto runtimeParametersNode = ConvertToNode(operation.RuntimeParameters);
+        if (runtimeParametersNode->GetType() == ENodeType::Map) {
+            if (auto acl = runtimeParametersNode->AsMap()->FindChild("acl")) {
+                return ConvertTo<TSerializableAccessControlList>(acl);
+            }
+            if (auto acoName = runtimeParametersNode->AsMap()->FindChild("aco_name")) {
+                return ConvertTo<TString>(acoName);
+            }
+        }
+    }
+    return {};
+}
```

Comment:
Let's reuse https://github.com/ytsaurus/ytsaurus/blob/85241756df65575c729debd1602b49491539abd7/yt/yt/ytlib/api/native/client_job_info_impl.cpp#L645 instead of this helper?

Reply:
We can customize this method to either fail or validate when it sees an empty ACL.

I am mostly against duplicating the ACR extraction logic.

Reply:
Hmm. Seems like validating against an empty ACL is fine since it will allow admins only and is equivalent to failing for regular users. Will use that